docs/docs/guides/web3_providers_guide/truffle.md

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_position: 6
+sidebar_label: 'Tutorial: Truffle'
+---
+
+# Tutorial: Truffle 
```

Comment:
I think the title `Tutorial: Truffle` is not indicative as Truffle is a framework.
So, if you will keep only the section related to Truffle Provider, then what about setting the title to something like `Tutorial: Use Truffle HDWallet Provider as an example of EIP 1193` or better: `Tutorial: Use a Third Party Provider` and then mention that `The Truffle HDWallet Provider will be used as an example of a third party provider that is EIP 1193 compatible.`

Reply:
The idea is to keep the title short so it can be only 1 line 
![image](https://github.com/web3/web3.js/assets/83420872/444a78f1-158b-428d-8060-b41f7459f14e)
I just put `Tutorial: Third Party provider` and then mentioned what u said above, in the first paragraph, i think is fine?